From 1ab302766bba1fba5ac23bbc9924cb00f3cd08d3 Mon Sep 17 00:00:00 2001
From: Greg Hackmann <ghackmann@google.com>
Date: Thu, 5 Feb 2015 16:54:00 -0800
Subject: [PATCH 565/634] android: base-cfg: enable ARMV8_DEPRECATED and
 subfeatures

ARMV8_DEPRECATED replaces the now-reverted ARMV7_COMPAT in the upstream
kernel

Change-Id: I5d5cee4e11c01d717692198fa070826930847703
Signed-off-by: Greg Hackmann <ghackmann@google.com>
---
 android/configs/android-base.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/android/configs/android-base.cfg b/android/configs/android-base.cfg
index bdc4749..d9857d2 100644
--- a/android/configs/android-base.cfg
+++ b/android/configs/android-base.cfg
@@ -6,6 +6,7 @@ CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_ANDROID_INTF_ALARM_DEV=y
 CONFIG_ANDROID_LOW_MEMORY_KILLER=y
+CONFIG_ARMV8_DEPRECATED=y
 CONFIG_ASHMEM=y
 CONFIG_BLK_DEV_DM=y
 CONFIG_BLK_DEV_INITRD=y
@@ -14,6 +15,7 @@ CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_DEBUG=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CGROUP_SCHED=y
+CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_DM_CRYPT=y
 CONFIG_DM_VERITY=y
 CONFIG_EMBEDDED=y
@@ -128,8 +130,10 @@ CONFIG_PREEMPT=y
 CONFIG_RESOURCE_COUNTERS=y
 CONFIG_RTC_CLASS=y
 CONFIG_RT_GROUP_SCHED=y
+CONFIG_SETEND_EMULATION=y
 CONFIG_STAGING=y
 CONFIG_SWITCH=y
+CONFIG_SWP_EMULATION=y
 CONFIG_SYNC=y
 CONFIG_SYSVIPC=y
 CONFIG_TUN=y
-- 
1.9.1

